src/profile/main/extensions/LINE-extension.js

```diff
@@ -0,0 +1,65 @@
+ if ( utag_data !== "undefined" && utag_data["device.device.type"].indexOf("Mobile") > -1 && (utag_data["dom.domain"].indexOf("www.expedia.co.jp") > -1 || utag_data["dom.domain"].indexOf("wwwexpediacojp") > -1))
```

Comment:
in case of safeframes, dom.domain will not give expedia domain. It will be safeframes domain